### PR TITLE
feat: integrations page

### LINKED ISSUE #20: Add integration page
**Description**
_Clearly and concisely explain the problem.
You can mention unclear or missing information, typos, grammatical mistakes, etc.
Any contributions and suggestions are welcome!_

"Discover integrations" page is missing, must be implemented as we now have the mockups

**Website Section URL**
_[Website live preview](https://quasar.beget.it), paste the page URL or section name (eg. Docs, if it happens on all docs page) below._

Docs

### LINKED ISSUE #81: Fix docs background color
**Description**
_Clearly and concisely explain the problem.
You can mention unclear or missing information, typos, grammatical mistakes, etc.
Any contributions and suggestions are welcome!_

Docs background color should be `#fafafa`
Currently it's plain white instead, which is color reserved to the drawers
This may also be the reason why the color contrast with the header seems so stark
Related #36 

**Website Section URL**
_[Website live preview](https://quasar.beget.it), paste the page URL or section name (eg. Docs, if it happens on all docs page) below._

Everywhere

### PR DESCRIPTION
**Pull request by n05la3**
Closes #20 
Closes #81

### CODE REVIEW

File: docs/src/components/landing-page/SearchQuasarForm.vue [hunk 1]
```diff
@@ -118,6 +118,10 @@ export default {
       emit('focus-by-keyboard', scope.focusByKeyboard.value)
     })
 
+    watch(() => scope.searchResults.value, () => {
```
Comment (IlCallo) [inline docs/src/components/landing-page/SearchQuasarForm.vue]: I think `watchEffect` would work in this case, to cleanup the code a bit

```ts
watchEffect(() => emit('search-result-change', scope.searchResults.value))
```

File: docs/src/router/routes.js [hunk 2]
```diff
@@ -8,6 +8,11 @@ const docsPages = [
     path: '',
     name: 'docPages',
     component: () => import('pages/Landing.vue')
+  },
+  {
+    path: 'integrations',
+    name: 'integrations',
+    component: () => import('pages/landing-page/QuasarIntegrations.vue')
```
Comment (IlCallo) [inline docs/src/router/routes.js]: This should be together with all other docs pages (I mean the file path), it's not like components one or the landing, which have it's own branding and management

File: docs/src/pages/landing-page/QuasarIntegrations.vue [hunk 3]
```diff
@@ -0,0 +1,175 @@
+<template>
+  <q-page class="column justify-center items-center bg-lp-light-grey q-px-xl font-monserrat">
```
Comment (IlCallo) [inline docs/src/pages/landing-page/QuasarIntegrations.vue]: All docs pages should have content bg as `bg-lp-light-grey`, as noted into https://github.com/dreamonkey/quasar/issues/81

Comment (IlCallo) [reply docs/src/pages/landing-page/QuasarIntegrations.vue]: We also need to raise the distance between content and the header for all docs pages, as noted into https://github.com/dreamonkey/quasar/issues/66

**Review by IlCallo — CHANGES_REQUESTED** [review]

File: docs/src/pages/landing-page/QuasarIntegrations.vue (hunk 3 above)
Comment (n05la3) [reply docs/src/pages/landing-page/QuasarIntegrations.vue]: Fixed #81, the fix for #66 in a subsequent PR.

**Pull request merged**